src/workerd/api/wpt/url-test.wd-test

```diff
@@ -0,0 +1,28 @@
+using Workerd = import "/workerd/workerd.capnp";
+
+const unitTests :Workerd.Config = (
+  services = [
+    ( name = "url-test-test",
+      worker = (
+        modules = [
+          (name = "worker", esModule = embed "url-test.js"),
+          (name = "url-origin.any.js",
+           esModule = embed "../../../../../wpt/url/url-origin.any.js"),
+          (name = "resources/urltestdata.json",
+           json = embed "../../../../../wpt/url/resources/urltestdata.json"),
+          (name = "resources/urltestdata-javascript-only.json",
+           json = embed "../../../../../wpt/url/resources/urltestdata-javascript-only.json"),
+        ],
```

Comment:
Each of the resources needed for the test end up needing to be embedded directly since we do not allow dynamic evaluation.

Reply:
Can we add *.wd-test inside this folder to gitignore and dynamically generate the config files?

Reply:
probably? I would need to buff up on my Bazel Kung Fu for that as my Bazel Kung Fu is weak.

Reply:
here is a good example: https://github.com/cloudflare/workerd/blob/main/src/workerd/server/tests/python/import_tests.bzl